tests/test_edtfdatestring.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow EDTF Date String."""
+
+import pytest
+from edtf.parser.grammar import ParseException, level0Expression
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import EDTFDateString
+
+
+class MySchema(Schema):
+    f = EDTFDateString()
```

Comment:
As fot the other PRs, it might be a good idea to set more descriptive names (e.g. `date`, or `edtfdate`) instead of placeholders.

same for `test_input` and `test_input_a`, they could be `date` or `edtfdate` too.